### PR TITLE
Update INSTALL.md

### PR DESCRIPTION
**Pull request by sahuang**
**What type of PR is this?**

documentation

**What this PR does / why we need it:**

Fix CentOS and update troubleshooting details

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -183,7 +185,7 @@ $ ./start_server.sh
 Follow the steps below to solve this problem:
 
 1. Make sure you have `libcurl4-openssl-dev` installed in your system.
-2. Try reinstalling the latest CMake from source with `--system-curl` option:
+2. Try to reinstall the latest CMake from source with `--system-curl` option:
```
Comment (yamasite) [inline INSTALL.md]: 这里的意思是尝试做，而不是努力做。不应改成to。另外，工程师请将注意力放在技术正确性上。

**Review by yamasite — CHANGES_REQUESTED** [review]
Please refer to my comment. Thanks.

File: INSTALL.md (hunk 1 above)
Comment (sahuang) [reply INSTALL.md]: ok

**Review by yamasite — APPROVED** [review]

File: INSTALL.md [hunk 2]
```diff
@@ -195,12 +197,18 @@ Follow the steps below to solve this problem:
 
 ### Error message: `internal compiler error`
 
-Try increasing the memory allocated to docker.
+Try to increase the memory allocated to Docker. If this doesn't work, you can reduce the number of threads in CMake build in `[Milvus root path]/core/build.sh`.
+
+```c++
```
Comment (yamasite) [inline INSTALL.md]: Should be shell

**Review by yamasite — APPROVED** [review]

**Review by yamasite — APPROVED** [review]

File: INSTALL.md [hunk 3]
```diff
@@ -199,8 +199,8 @@ Follow the steps below to solve this problem:
 
 Try to increase the memory allocated to Docker. If this doesn't work, you can reduce the number of threads in CMake build in `[Milvus root path]/core/build.sh`.
 
-```c++
-make -j 8 install || exit 1 // The default number of threads is 8.
+```shell
```
Comment (yamasite) [inline INSTALL.md]: Please add a dollar sign `$` before a shell command.

File: INSTALL.md [hunk 4]
```diff
@@ -199,8 +199,8 @@ Follow the steps below to solve this problem:
 
 Try to increase the memory allocated to Docker. If this doesn't work, you can reduce the number of threads in CMake build in `[Milvus root path]/core/build.sh`.
```
Comment (yamasite) [inline INSTALL.md]: Could you please also use **Try increasing** here? I just noticed this error. I might have overlooked this before.

**Review by yamasite — CHANGES_REQUESTED** [review]
Please refer to my latest comments.

**Review by yamasite — APPROVED** [review]

**Comment by yamasite** [comment]
This also fixes #1150 and #1151

**Pull request merged**